crowdin_api/api_resources/string_translations/resource.py

```diff
@@ -210,6 +210,7 @@ def list_string_translations(
         stringId: Optional[int] = None,
         languageId: Optional[str] = None,
         denormalizePlaceholders: Optional[DenormalizePlaceholders] = None,
+        taskId: Optional[int] = None,
```

Comment:
`list_string_translations` is the wrong method to add the `taskId` parameter. It should be `list_strings`:

https://github.com/crowdin/crowdin-api-client-python/blob/87e012220cc950f95216978609e7c2cd4cc07615/crowdin_api/api_resources/source_strings/resource.py#L33-L45

Reply:
I am extremely sorry! This is one of my first contributions to such a huge codebase. Will look into it. 
Also just so that i dont make any such mistakes, should i revert the changes in crowdin_api/api_resources/string_translations/tests/test_string_translations_resources.py? Your elaboration will be helpful

Reply:
No problem :)

Yes, it should be also moved to the respective test case: [`test_source_strings_resources.py`](https://github.com/crowdin/crowdin-api-client-python/blob/main/crowdin_api/api_resources/source_strings/tests/test_source_strings_resources.py)

Reply:
For reference, here is another example - https://github.com/crowdin/crowdin-api-client-python/pull/163

Reply:
Should I create a new PR? Or work on this by hard resetting the changes?

Reply:
No need to create a new PR, please do it within the current one